dnf/template-dnf-fedora.conf

```diff
@@ -12,19 +12,22 @@ color=never
 exclude=kernel
 overwrite_groups=0
 deltarpm=0
+zchunk=0
 
 #  This is the default, if you make this bigger yum won't see if the metadata
 # is newer on the remote and so you'll "gain" the bandwidth of not having to
 # download the new metadata and "pay" for it by yum not having correct
 # information.
 #  It is esp. important, to have correct metadata, for distributions like
 # Fedora which don't keep old packages around. If you don't like this checking
-# interupting your command line usage, it's much better to have something
+# interrupting your command line usage, it's much better to have something
 # manually check the metadata once an hour (yum-updatesd will do this).
 # metadata_expire=90m
 
+# PACKAGES_DIR is set by template builder plugin
```

Comment:
According to documentation, it can't be arbitrary name, it must have `DNF_VAR_` prefix:
> In addition to these hard coded variables, user-defined ones can also be used. They can be
  defined either via variable files, or by using special environmental variables. The  names
  of  these  variables  must be prefixed with DNF_VAR_ and they can only consist of alphanu‐
  meric characters and underscores:

Yum supported only `$YUM0` - `$YUM9` (and dnf still allow that) - that's why it was this name.